### DISCUSSION TITLE
How to join Slack workspace?

### DISCUSSION THREAD

**Discussion by llamafilm** [Haivision/srt#2716] [General]
The README in this repo suggests joining the Slack workspace at https://srtalliance.slack.com/. But when I try, it says this is only available to folks with an email from one of 3 approved domains. Is this meant to be public?
![D1A91149-702F-4A01-8715-B68D6791C62A](https://user-images.githubusercontent.com/3991046/231676247-e1cc57cf-5ed7-44ea-a0bc-117a6dacacda.jpeg)

**Comment by maxsharabayko** [comment]
There is also an invitation link in the README.
Although it is now quite detached from the place Slack is mentioned.
> To ask a question [join the conversation](https://slackin-srtalliance.azurewebsites.net/) ...

**Reply by maxsharabayko** [reply]
The link has been added to the table as well in #2721.

**Comment by llamafilm** [comment]
Thanks, I hadn't noticed that before.
